### DISCUSSION TITLE
制作的  m3u8 文件，同目录下的 ts 文件，alist 不能提供数据

### DISCUSSION THREAD

**Discussion by ferocknew** [AlistGo/alist#3318] [Q&A]
例如：

把视频切割成一个 m3u8，内容为：

```
#EXTM3U
#EXT-X-VERSION:3
#EXT-X-MEDIA-SEQUENCE:0
#EXT-X-ALLOW-CACHE:YES
#EXT-X-TARGETDURATION:11
#EXTINF:8.541667,
t-000.ts
#EXTINF:10.333333,
t-001.ts
#EXTINF:10.416667,
t-002.ts
#EXTINF:8.416667,
t-003.ts
#EXTINF:10.416667,
t-004.ts
#EXTINF:9.500000,
t-005.ts
#EXTINF:10.416667,
t-006.ts
#EXTINF:10.416667,
t-007.ts
#EXTINF:5.750000,
t-008.ts
#EXT-X-ENDLIST
```

对应的 ts 文件放在同文件夹下面。

<img width="776" alt="image" src="https://user-images.githubusercontent.com/2178683/217708112-2784c7a0-6bfc-4c2a-ab7a-fd43e8c61489.png">

然后，预览里添加 m3u8

<img width="466" alt="image" src="https://user-images.githubusercontent.com/2178683/217708176-50651c91-6aa7-45c9-911a-ab05788c3d9b.png">

m3u8 正常解析以后，ts 文件却没提供数据流

<img width="558" alt="image" src="https://user-images.githubusercontent.com/2178683/217708317-23be5f4e-82e9-47da-8c2a-8b9674c41599.png">

```
{"code":401,"message":"expire missing","data":null}
```

**Comment by voidfeng** [comment]
同样的问题，是签名的问题，开启了全局签名，ts文件不能直接访问，需要带上sign令牌才可以访问。不知道如何取消对文件.ts的签名
